hack/install-konveyor.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+set -x
+
```

Comment:
Perhaps you also want to add `set -o pipefail`? If a (pipeline) chained command fails, it only breaks the execution if the last command in the pipeline fails.